backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -333,12 +333,24 @@ public Project Update(Project project)
                 .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Fiscals)
                 .Include(p => p.Responses)
                 .SingleOrDefault(p => p.Id == project.Id) ?? throw new KeyNotFoundException();
-            this.ThrowIfNotAllowedToUpdate(originalProject, _options.Project);
 
             var userAgencies = this.User.GetAgencies();
             var originalAgencyId = (int)this.Context.Entry(originalProject).OriginalValues[nameof(Project.AgencyId)];
             if (!isAdmin && !userAgencies.Contains(originalAgencyId)) throw new NotAuthorizedException("User may not edit projects outside of their agency.");
 
+            //If the user isn't allowed to update the project, just update the notes.
+            if (!this.IsAllowedToUpdate(originalProject, _options.Project) && !project.IsProjectInDraft(_options.Project))
```

Comment:
For now we'll go with this, but it still has a security hole, as it allows for editing the note after a project is completed.  We'll have to readdress later.  Should add a / TODO